web_src/js/markdown/anchors.js

```diff
@@ -16,7 +16,7 @@ function scrollToAnchor() {
 }
 
 export default function initMarkdownAnchors() {
-  if (!document.querySelector('.markdown')) return;
+  if (!document.querySelector('.markup')) return;
```

Comment:
Probably also need to update `headingSelector` a few lines above. The fact that this change was necessary makes me suspicious thought, because I thought we'd have both classes present, e.g. `markdown markup`.

Reply:
Keeping both classes was my original proposition, but it was decided against because of duplication. https://github.com/go-gitea/gitea/pull/12261#issuecomment-708597615